example_project/example_project/orchestra_settings.py

```diff
@@ -68,11 +68,11 @@ def setup_orchestra(settings_module_name):
     # Tasks and Workflows
     ######################
 
-    # Installed orchestra workflows
-    settings.ORCHESTRA_PATHS = (
-        ('simple_workflow.workflow', 'simple_workflow'),
-        ('journalism_workflow.workflow', 'journalism_workflow'),
+    # Installed orchestra workflows to be included as Django apps
```

Comment:
I don't think we talk about the settings file anywhere other than in these comments—let me know if I'm wrong!